packages/react-router/index.ts

```diff
@@ -2,20 +2,34 @@ import * as React from "react";
 import type {
   ActionFunction,
   ActionFunctionArgs,
+  AgnosticRouteObject,
```

Comment:
I wonder if we need to keep exporting this type of stuff - we only really needed this because it was a standalone package - it could theoretically just use route objects now since it's all done from `react-router`